ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/controllerCore/WholeBodyInverseDynamicsSolver.java

```diff
@@ -200,40 +200,44 @@ public WholeBodyInverseDynamicsSolver(WholeBodyControlCoreToolbox toolbox, YoReg
       parentRegistry.addChild(registry);
    }
 
-   public void reset()
+   public void initialize()
```

Comment:
The diff here is pretty bad between this method and reset. I switched the two methods in the ordering of the class,  because reset is always called right before compute, and initialize is only called on request. However, the diff makes it look like there were a lot more changes than in reality.